src/app/libp2p_helper/src/codanet.go

```diff
@@ -129,13 +129,16 @@ type customValidator struct {
 // https://godoc.org/github.com/libp2p/go-libp2p-core/connmgr#ConnectionGating
 // the comments of the functions below are taken from those docs.
 type CodaGatingState struct {
+	logger       *logging.Logger
```

Comment:
Should this be `logging.EventLogger` instead?

Reply:
Wasn't using events, so I figured it didn't matter. Can change if you think it's better to leave the interface more general.